docs/maps/maps-getting-started.asciidoc

```diff
@@ -93,37 +94,38 @@ image::maps/images/gs_add_cloropeth_layer.png[]
 [[maps-add-elasticsearch-layer]]
 === Adding layers for {es} data
 
-You'll add two layers for {es} data. The first layer displays documents, and the
-second layer displays aggregated data.
-The raw documents appear when you zoom in the map to show smaller regions.
-The aggregated data 
-appears when you zoom out the map to show larger amounts of the globe.
+To avoid overwhelming the user with too much data at once, you'll add two layers for {es} data.
+
+. The first layer will show aggregated data that represents many documents.
```

Comment:
Make this a bulleted list and reverse the order to match the order of the sections in the doc.